Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -337,7 +337,7 @@
 #else
     auto image = RenderThemeMac::iconForAttachment(fileName, contentType, title);
 #endif
-    return convertPlatformImageToBitmap(image.get(), size);
+    return convertPlatformImageToBitmap(image.get(), iconSize);
```

Comment:
r=me

This function is pretty confusing to me. Might be worth a follow-up patch to clarify.

1. 'imageAndSize' and 'image' should be renamed to 'iconAndSize' and 'icon' since the type is IconAndSize and the data member is .icon.

2. 'size' should be part of the return value. It's super subtle that 'size' is an out parameter -- but only on one platform!

I wonder if we can test this with an API test?

Reply:
> This function is pretty confusing to me. Might be worth a follow-up patch to clarify.
> 
> 1. 'imageAndSize' and 'image' should be renamed to 'iconAndSize' and 'icon' since the type is IconAndSize and the data member is .icon.
> 2. 'size' should be part of the return value. It's super subtle that 'size' is an out parameter -- but only on one platform!
> 
> I wonder if we can test this with an API test?

Yes, absolutely, I have created https://bugs.webkit.org/show_bug.cgi?id=242269 to track this.

Thanks for reviewing!